version.go

```diff
@@ -25,7 +26,9 @@ type versionsByNumber []*Version
 func (s versionsByNumber) Len() int      { return len(s) }
 func (s versionsByNumber) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s versionsByNumber) Less(i, j int) bool {
-	return s[i].Number < s[j].Number
```

Comment:
It looks like we already have weakly-typed input. If you change the type on that struct to an int (or uint) instead of string, it should "just work". Not sure what to do about BC though.